src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -82,17 +86,29 @@ export const SykmeldingTittelbeskrivelse = ({
   const erViktigInformasjon = erEkstraInformasjonISykmeldingen(sykmelding);
   const sykmelder = sykmelding.bekreftelse.sykmelder;
   const arbeidsgiver = arbeidsgivernavnEllerArbeidssituasjon(sykmelding);
+  const erIkkeTattIBruk = sykmelding.status === SykmeldingStatus.NY;
 
   return (
     <div className="w-full flex flex-col">
-      <div className="flex justify-between">
+      <div className="flex justify-between mb-2">
         <div>
           {periode}
           {graderinger}
         </div>
-        {erViktigInformasjon && (
-          <img alt="Viktig informasjon" src={MerInformasjonImage} />
-        )}
+        <div className="flex gap-4">
+          {erIkkeTattIBruk && (
+            <Tag variant="warning" size="small">
+              {texts.sykmeldinger.ny}
+            </Tag>
+          )}
+          {erViktigInformasjon && (
+            <img
+              height={18}
```

Comment:
Hvis skjermen er liten og Tag-en blir på to linjer, så følger ikonet med de tre prikkene størrelsen etter tagen, i stedet for en fixed height. Så fikset dette.